docs/en_us/internal/testing.rst

```diff
@@ -281,7 +281,7 @@ console::
     paver test_js_run -s common
     paver test_js_run -s common-requirejs
 
-To run JavaScript tests in a browser:
+To run JavaScript tests in a browser::
```

Comment:
Agreed, @vkaracic it is a common practice. As you saw in my comment to @singingwolfboy we are trying to move toward a writing style that is easier to understand, faster and less expensive to translate. No need to go through the whole file, the doc team is working to make those types of consistency changes (though it is a slow process).  We're okay with inconsistencies as this will take some time.